### PR TITLE
Suggest/dev doc fixes

### PR DESCRIPTION
**Pull request by ashander**
Great docs! I've provided some small fixes, feedback and suggestions. Let me know if anything doesn't make sense

See `docs/feedback.txt` for additional notes.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
This is great, thanks @ashander! Would you mind squashing the commits down please? I'll follow with a commit then to respond to the queries and tidy up the loose ends.

**Comment by ashander** [comment]
No problem. They're squashed

File: docs/feedback.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+requirements.txt`)
+
+2. nosetests gave some failures without numpy and h5py installed (noted in PR)
+
+3. running `make` in the `lib` directory worked to build the c cli
+
+4. after install, running `msp` or `mspms` from the command line gave me an
+error `pkg_resources.DistributionNotFound: The 'svgwrite' distribution was not
+found and is required by msprime` is there some additional sysreq for svgwrite?
+It appears with `pip freeze` and I can load it fine in python.
```
Comment (ashander) [inline docs/feedback.txt]: This was apparently due to a system-level install of `msprime` interfering with my version (installed locally using `pip -e .`) when I build and run locally, as now explicitly suggested, things work fine

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,10 @@
 codecov
 coverage
 flake8
+h5py
```
Comment (jeromekelleher) [inline requirements.txt]: Ah, I remember now why I didn't include h5py and numpy in the requirements.txt --- it caused issues on readthedocs.

I think we'll need a few different requirements.txt files, hopefully being able to 'source' each other including more-and-more specific requirements.

**Comment by jeromekelleher** [comment]
Thanks for the updates @ashander. I think we need a slightly different setup for the requirements.txt files. I'll push some changes here in a few days --- I'd prefer not to break the RTD build by merging this now.

**Comment by jeromekelleher** [comment]
Merging this now, going to follow up with another PR tweaking the requirements.txt infrastructure. Thanks again @ashander!

**Pull request merged**